safe/definitions/post_processors/__init__.py

```diff
@@ -1,9 +1,11 @@
 # coding=utf-8
-from population_post_processors import *
-from post_processors import *
+from safe.definitions.post_processors.productivity_post_processors import \
+    productivity_post_processors
```

Comment:
__[flake8]__
* [F403] 'from population_post_processors import *' used; unable to detect undefined names

<!-- Comment by [SideCI](https://sideci.com) -->